governance/votes/Draft- Executive vote - September 13, 2023.md

```diff
@@ -0,0 +1,218 @@
+date: 2023-09-13T00:00:00.000Z
+address: "$spell_address"
+
+
+
+
+
+---
+
+# [Executive Proposal] Scope-Defined Parameter Changes, Spark Protocol D3M Parameter Changes, Delegate Compensation, Set Fortunafi Debt Ceiling to Zero, DAO Resolution for HV Bank and Other Actions - September 13, 2023
+
+The Reserve Governance Facilitators, Sidestream, Dewiz and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following actions within the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+
+- Several Stability Scope-defined parameter changes will be executed, as detailed below.
+- The Spark Protocol Dai Direct Deposit Module (D3M) Maximum Debt Ceiling will be increased to 400 million DAI; and the Ceiling Increase Cooldown will be increased to 12 hours. 
+- Aligned Delegate compensation for August 2023 will be distributed.
+- The Fortunafi vault's Debt Ceiling will be set to zero DAI, per the Stability Scope Artifact's requirement to offboard legacy Legal Recourse Assets.
+- A DAO resolution pertaining to the HV Bank vault will be approved to terminate future purchases and return available cash from the HVB Master Participation Trust. This is per the Stability Scope Artifact's requirement to offboard legacy Legal Recourse Assets.
+- A Spark Protocol Proxy Spell will be triggered that implements changes to the DAI borrow spread and the flash loan fee.
+- MKR vesting transfers will be executed for the DECO-001 and SES-001 Core Units, per their successful budget proposals. 
+- The `MCD_CAT` contract will be scuttled by removing it from the Chainlog and removing all its permissions on the Protocol contracts.  
+
+**Voting for this executive proposal will place your MKR in support of the actions outlined above.**
+
+Unless otherwise noted, the actions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the actions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+For voters' convenience, each proposed item is prefaced with information showing its provenance. 
```

Comment:
I believe this section is redundant; our goal should be to maintain concise and succinct copy. This information should be readily deducible by the typical spell voter.

Reply:
I agree, I don't think we need to explain this

Reply:
However, I really like the idea of adding Authorization, Reasoning, and Authorizing Party to each item

Reply:
Thanks JS - please see discussion in our Discord channel re: this.

Reply:
Thanks VW & JS

I'm going to leave in this introductory info to the new formatting.

I don't some things are obvious or readily deducible. If we want to be welcoming to people who are not within the inner circle, for instance, it is not obvious what "authorization" pertains to. What sort of sources of authorization are valid in the Maker system? Someone relatively new to MKR governance - and we'll likely be getting a larger and larger stream of new people - will not know the answer to this.

Same goes with Authorizing Parties. Yes, the meaning of the phrase is self evident, but who counts as an authorizing party? To specifically list the possibilities of Responsible Scope Facilitators or advisory council members informs new participants about critical stakeholders.